addons/web/static/src/views/graph/graph_arch_parser.js

```diff
@@ -65,6 +65,9 @@ export class GraphArchParser {
                         archInfo.fieldAttrs[fieldName].isInvisible = true;
                         break;
                     }
+                    if (node.hasAttribute("widget")) {
```

Comment:
would be nice to do like for attribute string and put everything in fieldAttrs.
It would reduce you diff